04/app.js

```diff
@@ -0,0 +1,33 @@
+const divList = document.querySelectorAll('[data-time]');
+
+
+function addClass (e) {
+  const element = e.currentTarget
+  // e.stopPropagation();
+  setTimeout(function () {
+    element.classList.add('clicked');
+  }, element.dataset.time)
+  
+}
+
+
+
+function removeClass () {
```

Comment:
Spróbuj tak zrobić, aby zawartość funkcji była uruchamiana jedynie jak się kliknie poza divy ;)
Obecnie jak klikasz w div to też się uruchamia, ale nie widać efektu bo jest wykonywana od razu jak jeszcze nie ma odpowiednich klas